disperser/cmd/controller/main.go

```diff
@@ -148,7 +149,20 @@ func RunController(ctx *cli.Context) error {
 			return err
 		}
 	}
-	nodeClientManager, err := controller.NewNodeClientManager(config.NodeClientCacheSize, logger)
+
+	var requestSigner clients.RequestSigner
+	if !config.DisperserStoreChunksSigningDisabled {
```

Comment:
if the store chunks signing is disabled, we should probably throw a warning on the start up logs.

Reply:
var requestSigner clients.DispersalRequestSigner
	if config.DisperserStoreChunksSigningDisabled {
		logger.Warn("StoreChunks() signing is disabled")
	} else {
		requestSigner, err = clients.NewDispersalRequestSigner(
			context.Background(),
			config.AwsClientConfig.Region,
			config.AwsClientConfig.EndpointURL,
			config.DisperserKMSKeyID)
		if err != nil {
			return fmt.Errorf("failed to create request signer: %v", err)
		}
	}